Tests/PSModuleGallery.Type.Tests.ps1

```diff
@@ -161,6 +161,51 @@ InModuleScope 'PSDepend' {
             }
```

Comment:
Note: I wanted to make changes to previous context/test ([line 154-156](https://github.com/RamblingCookieMonster/PSDepend/pull/61/files#diff-111bd159e47a2144c2ba7679ec36c128R154)) and test but didn't to avoid merge conflicts with PR https://github.com/RamblingCookieMonster/PSDepend/pull/58.

Once PR 58 is merged we should then change:

``` powershell
Context 'Misc'
  It 'Adds folder to path when specified' {
    Mock Get-PSRepository { Return $true }
    Mock Save-Module {$True}
```

... to:
``` powershell
Context 'AddToPath on install of module to target folder' {
  It 'Adds folder to path' {
    Mock Save-Module {$True}
```

Reply:
Hiyo!  Sorry for taking so long - Want to rebase or merge that?

Thanks!

Reply:
I've made the change and rebased on top of master